### DISCUSSION TITLE
with all examples unable to resolve %any, initiate aborted

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [strongswan/strongswan#2127] [Q&A]
With all examples[1] I get the error message. 
```
test2:/etc/swanctl/conf.d# swanctl --initiate --child win
15[CFG] vici initiate CHILD_SA 'win', me (null), other (null), limits 0
16[IKE] unable to resolve %any, initiate aborted
[IKE] unable to resolve %any, initiate aborted
initiate failed: establishing CHILD_SA 'win' failed
test2:/etc/swanctl/conf.d#
```

on alpine linux strongswan-5.9.12
[1]
https://docs.strongswan.org/docs/5.9/interop/windowsMachineServerConf.html

**Comment by tobiasbrunner** [comment]
Obviously, these are server configs for roadwarrior clients. You can't initiate them actively as there is no remote endpoint. They have to be initiated by a client (as the error message says, the remote address is set to `%any` to accept clients from arbitrary IP addresses).

**Comment by f1-outsourcing** [comment]
Yes it is not really clear even how to start strongswan (as if you have systemd in a tiny container)

**Reply by tobiasbrunner** [reply]
Just use and start charon in a container, not charon-systemd.
